cvprac/cvp_client.py

```diff
@@ -660,17 +660,18 @@ def _make_request(self, req_type, url, timeout, data=None,
         if response:
             try:
                 resp_data = response.json()
+            except JSONDecodeError as error:
+                self.log.error(error)
+                raise error
             except ValueError as error:
-                self.log.debug('Error trying to decode request response %s',
-                               error)
                 if 'Extra data' in str(error):
                     self.log.debug('Found multiple objects in response data.'
                                    'Attempt to decode')
                     decoded_data = json_decoder(response.text)
                     resp_data = dict(data=decoded_data)
                 else:
-                    self.log.debug('Attempt to return response text')
-                    resp_data = dict(data=response.text)
+                    self.log.error(error)
+                    raise error
```

Comment:
This breaks handling of Resource API endpoints that return multiple JSON objects.

The specific format that isn't being handled is below and it was likely previously masked by the response.text.

Bad Format not being handled:

[Errno Expecting value] : 0

Good format:

[Errno Extra data] {"result":{"value":{"key":{"workspaceId":"CVPRAC_TEST_b7e898b8-dba8-11ec-ba4c-a683e7158098","elementType":"ELEMENT_TYPE_DEVICE","label":"cvpractestdev","value":"TAGTESTDEV"},"remove":false},"time":"2022-05-24T21:30:23.853711633Z","type":"INITIAL"}}
{"result":{"value":{"key":{"workspaceId":"CVPRAC_TEST_b7e898b8-dba8-11ec-ba4c-a683e7158098","elementType":"ELEMENT_TYPE_INTERFACE","label":"cvpractestint","value":"TAGTESTINT"},"remove":false},"time":"2022-05-24T21:30:23.710580226Z","type":"INITIAL"}}
: 249